log-manager/src/main/java/io/airlift/log/StaticFormatter.java

```diff
@@ -43,6 +50,16 @@
             .appendOffset("+HHMM", "Z")
             .toFormatter(Locale.US);
 
+    public StaticFormatter(Map<String, String> logAnnotations)
+    {
+        this.logAnnotations = ImmutableMap.copyOf(requireNonNull(logAnnotations, "logAnnotations is null"));
+    }
+
+    public StaticFormatter()
```

Comment:
Move this above and chain to other constructor
```java
this(ImmutableMap.of());
```
It's preferable to have everything go through a single constructor if possible